django/contrib/auth/forms.py

```diff
@@ -266,7 +272,7 @@ class SetPasswordForm(forms.Form):
     }
     new_password1 = forms.CharField(label=_("New password"),
                                     widget=forms.PasswordInput,
-                                    help_text=password_validation.password_validators_help_text_html())
+                                    help_text=mark_safe(password_validation.password_validators_help_text_html()))
```

Comment:
This should be done directly in https://github.com/django/django/blob/master/django/contrib/auth/password_validation.py#L87 (maybe use `format_html_join`)